include/kernels/block_orthogonalize.cuh

```diff
@@ -63,7 +63,7 @@ namespace quda {
     template <typename... T>
     BlockOrthoArg(ColorSpinorField &V, const int *fine_to_coarse, const int *coarse_to_fine, int parity,
                   const int *geo_bs, const int n_block_ortho, const ColorSpinorField &meta, T... B) :
-      kernel_param(dim3(meta.VolumeCB() * (fineSpin > 1 ? meta.SiteSubset() : 1), chiral_blocks, 1)),
+      kernel_param(dim3(meta.VolumeCB() * (fineSpin > 1 ? meta.SiteSubset() : 1), 1, chiral_blocks)),
```

Comment:
Out of curiosity, what type of performance difference does this make?

Reply:
None.  It's a cleanup really for consistency with the multi-reductions where the batched index is in the z dimension.

It's a bit more consequential in the restrictor, where we actually have a batch size greater than 1 in the block.  There we were mapping the y thread index to the batch index, which is the z index in the block reducer.  It didn't break anything, or rely on UB, but it gave me nightmares, e.g., if CUB changed its internals somehow, it could break.  So making this all consistent was a good thing to do.